src/components/Turns/Turns.js

```diff
@@ -4,24 +4,38 @@ import images from "../../assets/images";
 
 const { RedBullet, BlueBullet } = images;
 
-const Turns = ({ turn, player1Pos, player2Pos }) => {
-  // This function will create the appropriate classnames for the bullets sitting next to each player's name.
-  const bulletClassName = (turnNum) => {
-    return classNames({
-      bullet: true,
-      visible: turn === turnNum,
-      hidden: turn !== turnNum,
-    });
-  };
-
-  // This function will create the appropriate classnames for the player's information div.
-  const playerInfoClassName = (turnNum) => {
-    return classNames({
-      playerInfo: true,
-      bold: turn === turnNum,
-    });
-  };
+// You could make these util functions pure functions
+// and move these functions outside of the component
+// or into a util
+
+// This function will create the appropriate classnames for the bullets sitting next to each player's name.
+const bulletClassName = (turn, turnNum) => {
+  return classNames({
+    bullet: true,
+    visible: turn === turnNum,
+    hidden: turn !== turnNum,
+  });
+};
+
+// This function will create the appropriate classnames for the player's information div.
+const playerInfoClassName = (turn, turnNum) => {
+  return classNames({
+    playerInfo: true,
+    bold: turn === turnNum,
+  });
+};
+
+// You could also compose them into 1 function
+// which will return you a tuple of classes:
+//
+// const getClasses = (turn, turnNum) => {
+//   const bulletClass = bulletClassName(turn, turnNum);
+//   const playerClass = playerInfoClassName(turn, turnNum);
 
+//   return [bulletClass, playerClass];
```

Comment:
I have chosen an array here because I know I am returning a tuple.
When you use `React.useState()` it return you a tuple as well: `[state, setState]`

You can destructure the tuple and use their values the same way you would do with `React.useState()`:
```js
const [firstClass, secondClass] = getClasses(x, y, z)
```

If you prefer an object, you can simply return an object, there is no right or wrong here.

In this scenario, I prefer the tuple approach. I can easily name these values anything during destructuring, when it comes to an object, you are a bit restricted with the names. But as I mentioned it before, it is comes down to you personal preferences :)